Engine/DataFeeds/LiveTradingDataFeed.cs

```diff
@@ -539,6 +577,44 @@ private Subscription CreateUniverseSubscription(SubscriptionRequest request)
                 });
                 enumerator = enqueable;
             }
+            else if (request.Universe is OptionChainUniverse)
```

Comment:
Similar comment on sprawling conditionals as above... just something to think about for future refactorings and simplification.